llvm/Maintainers.md

```diff
@@ -305,10 +305,10 @@ jakub@nod-labs.com (email), [kuhar](https://github.com/kuhar) (GitHub)
 Peter Collingbourne \
 peter@pcc.me.uk (email), [pcc](https://github.com/pcc) (GitHub)
 
-#### CMake and library layering
+#### CMake
 
-Chandler Carruth \
-chandlerc@gmail.com, chandlerc@google.com (email), [chandlerc](https://github.com/chandlerc) (GitHub)
+Petr Hosek \
+phosek@google.com (email), [petrhosek](https://github.com/petrhosek) (GitHub)
```

Comment:
@petrhosek is a fantastic choice for CMake maintainer, but I actually think we probably need more than one maintainer for CMake. LLVM's CMake has really become extremely complex and varied, and there are a lot of changes that go into it (big and small).

I had started and never followed through on creating a set of code owners for CMake which at the time was @petrhosek, @compnerd, @smeenai, @ldionne, @tstellar, and myself. I'd probably add @MaskRay to that list too.